frontend/src/main/components/Commons/CommonsCard.js

```diff
@@ -8,7 +8,7 @@ const CommonsCard = (props) => {
                 <Row>
                     <Col sx={4} data-testid="commonsCard-id">{props.commons.id}</Col>
                     <Col sx={4} data-testid="commonsCard-name">{props.commons.name}</Col>
-                    <Col sm={4}>{props.buttonText != null && <Button data-testid={`commonsCard-button-${props.buttonText}-${props.commons.id}`} variant={'danger'} size="sm" className="mx-4" onClick={() => props.buttonLink(props.commons.id)} >{props.buttonText}</Button>}</Col>
+                    {props.buttonText != null && <Col sm={4}><Button data-testid={`commonsCard-button-${props.buttonText}-${props.commons.id}`} variant={'danger'} size="sm" className="mx-4" onClick={() => props.buttonLink(props.commons.id)} >{props.buttonText}</Button></Col>}
```

Comment:
Horrific hack of the `CommonsList` component, abusing JavaScript Boolean expressions that yield not truth values but their operands. To properly implement the public-facing commons list we need to add a new backend method. We should prioritize the administrative commons list first and circle back if we have time.

Reply:
Using `||` and `&&` for this is common enough within the React ecosystem. Until JS gets do-expressions, I think this is fine.

Reply:
Agreed with Bryan.... it's weird looking at first, but pretty common among React programmers.